src/jestTransformer.ts

```diff
@@ -0,0 +1,64 @@
+import { createHash } from 'crypto';
+import { Transformer } from '@jest/transform';
+import { createTransformer, getCacheKey as getBabelCacheKey } from 'babel-jest';
+import graphQLTransformer from 'jest-transform-graphql';
+import { relative as pathRelative } from 'path';
+import { readFileSync } from 'fs';
+import loadConfig from './lib/load-config';
+import { createPaths } from './lib/paths';
+
+const isSchemaFile = (filePath: string) => filePath.endsWith('.graphqls');
```

Comment:
Problem is the extension is up to users. Maybe we can pick it up from `schema` field in `.graphql-let.yml`.

Reply:
Good idea, I changed to do that.